tests/Tests.AzureAppConfiguration/RefreshTests.cs

```diff
@@ -920,6 +833,33 @@ void action() => new ConfigurationBuilder()
             Assert.Throws<ArgumentException>(action);
         }
 
+        [Fact]
+        public void RefreshTests_CacheExpiresIsNotDefaultValue()
```

Comment:
This seems like an implementation detail that is covered by the other test you added.